apriltag.c

```diff
@@ -445,7 +445,9 @@ static matd_t* homography_compute2(double c[4][4]) {
             }
         }
 
-        assert(max_val_idx >= 0);
+        if (max_val_idx < 0) {
```

Comment:
Optional: This condition could be combined with the `max_val < epsilon` check below